3.2/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -259,3 +259,59 @@ function setup_keyfile() {
 function info() {
   printf "=> [%s] %s\n" "$(date +'%a %b %d %T')" "$*"
 }
+
+# insert_and_wait_for_replication insert data in host and wait all replset members
```

Comment:
since the two scripts you're adding are only used from test code, it seems a bit confusing to put them in the image itself.  can you move them to a common test util script instead/